multiqc/templates/default/content.html

```diff
@@ -10,41 +10,46 @@
 {% for m in report.modules_output %}
   {% if m.sections | length > 0 %}
   <div id="mqc-module-section-{{ m.anchor }}" class="mqc-module-section">
-    <h2 id="{{ m.anchor }}">{{ m.name }}</h2>
-    {{ m.intro if m.intro }}
-    {% if m['comment'] %}<blockquote class="mqc-section-comment">{{ m['comment'] }}</blockquote>{% endif %}
-    {% for s in m.sections %}
-      {% if s['print_section'] %}
-        {% if (s['name'] is none or s['name'] | length == 0) and s['helptext'] is not none and s['helptext'] | length > 0 %}
-          <button class="btn btn-default btn-sm pull-right" type="button" data-toggle="collapse" data-target="#{{ s['anchor'] }}_helptext" aria-expanded="false" aria-controls="{{ s['anchor'] }}_helptext">
-            <span class="glyphicon glyphicon-question-sign" aria-hidden="true"></span>
-            Help
-          </button>
-        {% endif %}
-        <div class="mqc-section mqc-section-{{ m.anchor }}">
-          {% if s['name'] is not none and s['name'] | length > 0 %}
-            <h3 id="{{ s['anchor'] }}">
-                {{ s['name'] }}
-                {% if s['helptext'] is not none and s['helptext'] | length > 0 %}
-                  <button class="btn btn-default btn-sm pull-right" type="button" data-toggle="collapse" data-target="#{{ s['anchor'] }}_helptext" aria-expanded="false" aria-controls="{{ s['anchor'] }}_helptext">
-                    <span class="glyphicon glyphicon-question-sign" aria-hidden="true"></span>
-                    Help
-                  </button>
-                {% endif %}
-            </h3>
-          {% endif %}
-          {% if s['description'] is not none and s['description'] | length > 0 %}<div class="mqc-section-description">{{ s['description'] }}</div>{% endif %}
-          {% if s['comment'] is not none and s['comment'] | length > 0 %}<blockquote class="mqc-section-comment">{{ s['comment'] }}</blockquote>{% endif %}
-          {% if s['helptext'] is not none and s['helptext'] | length > 0 %}
-            <div class="collapse mqc-section-helptext " id="{{ s['anchor'] }}_helptext">
-              <div class="well">{{ s['helptext'] }}</div>
-            </div>
+    <div class="mqc-module-section-first">
```

Comment:
Do we not need a conditional here as well, to mirror the closing `</div>`?

```suggestion
    {% if loop.first %}
    <div class="mqc-module-section-first">
    {% endif %}
```

Reply:
No. This is a bit hacky, but we're actually grouping the `h2` and first loop iteration into a single `div`. This means that the `h2` and the section right after wouldn't be split across a page break (with the right css classes).

If you'd prefer, I could move:
```
      <h2 id="{{ m.anchor }}">{{ m.name }}</h2>
     {% if m['comment'] %}<blockquote class="mqc-section-comment">{{ m['comment'] }}</blockquote>{% endif %}
```

Right into the second for, and we could guard them with `if loop.first`. I think that's slightly cleaner. Thoughts?

Reply:
Ah right, sorry I missed that this was outside the loop. Yeah, might be nicer if the loop start is moved up above this chunk as you say. Might help future-me some pain 😅